test/extended/prometheus/prometheus_builds.go

```diff
@@ -38,11 +38,16 @@ var _ = g.Describe("[Feature:Prometheus][Feature:Builds] Prometheus", func() {
 
 	g.Describe("when installed on the cluster", func() {
 		g.It("should start and expose a secured proxy and verify build metrics", func() {
+			if exutil.IsClusterOperated(oc) {
+				// TODO: prometheus is not scraping build metrics on 4.0 installer clusters... something for the prometheus team
+				// to dig into?
```

Comment:
Please make sure there is an open bug here.

Reply:
https://jira.coreos.com/browse/DEVEXP-235